### DISCUSSION TITLE
Skill level

### DISCUSSION THREAD

**Discussion by anchillabc** [bananarama92/MBS#33] [Q&A] [answered]
Is the application level of the stuff applied by the wheel based off the total or current bondage skill level of the wheel owner?

**Comment by bananarama92** [comment]
It's based on the player's bondage skill, so it uses the same logic that's used for self bondage when equipping a conventional restraint.

**Reply by anchillabc** [reply]
So, as a wheel operator, I have no control over how difficult it is going to be for other players to escape when they spin my wheel?

**Reply by bananarama92** [reply, marked as answer]
Besides the ability to mandate locks (or not) or to use crafted items with the `Secure` property? No you don't, wheel spins are more so treated as a form of self bondage by MBS.

**Reply by anchillabc** [reply]
Thank you, that explains the inconsistent results I got when trying to understand why some people could never get out of stuff I crafted with the `Loose` property...
